### ISSUE TITLE
[Quest/Object] Waste Not, Want Not

### ISSUE THREAD

**Issue by LightningBlade** [NetherstormTeam/IssueTracker#30]
Quest Link - http://tbc.openwow.com/quest=10055

Object Link - http://tbc.openwow.com/object=182938

The Object is supposed to immediately disappear upon looted, however it still appears visible ( Sparkling ) for 27 - 30 seconds after being looted

**Comment by Clearlysql** [comment]
Did you try this with .gm off?

**Comment by LightningBlade** [comment]
Of course, I did it as a player would, and one of the Testers also reported this happening too through skype. 

The Wood works perfectly fine, but the Metal would remain on the ground from approximately 30 seconds, and have the small light sparks around it.

**Comment by Clearlysql** [comment]
I have tested a handful of Salvageable Metal and all of them work properly, despawns for players as they should but are still around when you have your GM tag on, as they should. Nothing of this seems wrong. Could you test this again and see if you can reproduce this and write down exactly where it is? If possible, do provide the GUID for the objects this occur with as well.

Thank you.

**Comment by Clearlysql** [comment]
Client side issue. Should work fine if a new client is installed as it works fine for the others who tested it.

**Comment by LightningBlade** [comment]
Reproduced last night, as seen by yourself... I think it's low priority as assigned by Jamey
